modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v1/structure.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+require 'prawn/table'
+
+module AppealsApi
+  module PdfConstruction
+    module NoticeOfDisagreement::V1
+      class Structure
+        def initialize(notice_of_disagreement)
+          @notice_of_disagreement = notice_of_disagreement
+        end
+
+        # rubocop:disable Metrics/AbcSize
+        def form_fill
+          options = {
+            form_fields.veteran_name => form_data.veteran_name,
+            form_fields.veteran_ssn => form_data.veteran_ssn,
+            form_fields.veteran_file_number => form_data.veteran_file_number,
+            form_fields.veteran_dob => form_data.veteran_dob,
+            form_fields.mailing_address_number_and_street => form_data.mailing_address_number_and_street,
+            form_fields.homeless? => form_data.homeless?,
```

Comment:
Do the `form_fields` predicate methods indicate a checkbox type field?

Reply:
Essentially, yes.